### ISSUE TITLE
Joomla 5.1.x - If intro image name (url) contains a blank the img src in rss feed is invalid

### ISSUE THREAD

**Issue by coolcat-creations** [joomla/joomla-cms#43742]
### Steps to reproduce the issue
Upload and assign an image with blanks to an article
Upload and assign an image without blanks to an article
Access the rss feed to the category blog or featured articles view where this image should be included.

### Expected result
Both images should appear in the feed (actually in an <enclosure> Tag - but thats another topic - they should at least appear in the description tag

### Actual result
the image with the blanks is not rendered correctly but with an invalid src

### System information (as much as possible)
Joomla 5.1.1
Featured and Blog View

**Comment by brianteeman** [comment]
What do you mean by "an image with blanks"

**Comment by coolcat-creations** [comment]
@brianteeman Image with a title with blanks, for example: this is an image.jpg
It will result in this%20is%20an%20image.jpg as a value and also can be retrieved and is shown in the blog but the feed wouldn't render it anyway

**Comment by brianteeman** [comment]
just to be clear as there is a translation issue here. You are saying that the problem is with a file that has spaces in the filename

**Comment by coolcat-creations** [comment]
Yes, if the image file name has spaces (I thought it's called blanks) in the file name the rss feed does not show the correct url in the image but just the root url.

**Comment by brianteeman** [comment]
Thanks for clarifying that. I can confirm that an image with spaces in the filename is not included in the source of the rss feed. I assume that this is what you are seeing

Without spaces 
`<description><![CDATA[<div class="feed-description"><p><img src="https://brianstest.site//images/filename.jpg" alt="Boy in blue ski jacket holding a big pile of snowballs"></p>`

With spaces
`<description><![CDATA[<div class="feed-description"><p><img src="https://brianstest.site/" alt="Boy in blue ski jacket holding a big pile of snowballs"></p>`

**Comment by coolcat-creations** [comment]
Correct - also double slashes on the correct ones

**Comment by level420** [comment]
This bug is also present in Joomla 4 up to version 4.4.9.
